container/rules/debian_pkg_tar.bzl

```diff
@@ -0,0 +1,188 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@base_images_docker//package_managers:download_pkgs.bzl", _download_deb_pkgs = "download")
+load("@base_images_docker//package_managers:apt_key.bzl", _apt_key = "key")
+
+def _input_validation(kwargs):
+    allowed_attribues = ["name", "base", "language_layers"]
+
+    for key in kwargs.keys():
+        if key not in allowed_attribues:
+            fail("Attribute " + key + " is not supported.")
+
+container = [
+    ".tar.gz",
+    ".tgz",
+    ".tar",
+    ".tar.xz",
+]
+
+debian_pkgs_attrs = _apt_key.attrs + _download_deb_pkgs.attrs + {
```

Comment:
using _apt_key.attrs (https://github.com/GoogleContainerTools/base-images-docker/blob/master/package_managers/apt_key.bzl#L97) drags in all the attrs from container_image. Lets just list out exactly which attrs we want to allow here.

Reply:
Yea, I will only list base, packages, and keys here.